src/components/Home/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from "react"
+
+type Props = Readonly<{ children?: React.ReactNode }>
+
+export function Home({ children }: Props) {
+  const [count, setCount] = useState(0)
```

Comment:
👍 🎉 I like that you used hooks here!

Reply:
Awesome! However, this was scaffolding from when running the new:component command. But heck, I'll take the thumbs up anyway!